### PR TITLE
Add timeout to healthcheck curl

### PR DESCRIPTION
**Pull request by Autophagy**

### CODE REVIEW

**Comment by Autophagy** [comment]
This should address #113. Not sure why the tests are failing here.

File: Dockerfile.template [hunk 1]
```diff
@@ -56,7 +56,7 @@ ENV CRATE_HEAP_SIZE 512M
 # if we cannot get any response from the CrateDB (connection refused, timeout
 # etc). If any response is received (regardless of http status code) we
 # consider the node as running.
-HEALTHCHECK CMD curl $(hostname):4200
+HEALTHCHECK --timeout=35s --interval=35s CMD curl -f --max-time 30 $(hostname):4200
```
Comment (chaudum) [inline Dockerfile.template]: `--fail` if you want to be consistent with 3.0 template

**Comment by chaudum** [comment]
Personally I would trigger the Docker healthcheck every 30sec and make the curl timeout shorter than 30sec (e.g. only 15-20sec), but doing it like that is totally ok!

**Comment by Autophagy** [comment]
Okidoke ! wasn't too sure as to which way around I should put the timeouts

**Pull request merged**